content/posts/newsletter/2020-09-06.md

```diff
@@ -24,6 +24,8 @@ _To everyone involved, please do not just drop links, rather try to make a short
 formulated description of what is the main outcome of your source. This will help us a lot! 
 You do not need to use a formal tone, be passionate about why this motivates you or what you work on!_
 
+:de: The German blog kuketz-blog.de published [part 6 of it's messenger series](https://www.kuketz-blog.de/conversations-messaging-ueber-das-xmpp-protokoll-messenger-teil6/) covering the Android client Conversations.
```

Comment:
@Echolon 
`:de:` was not rendered correctly in HTML last time it was used and is an accessibility issue: I suggest we stick to a simple `[<language>]` notation next to the link or at the end or beginning of the paragraph

Reply:
I suggest we just leave it away. People will be fine and they can use Google translate I think.

Ok?

Reply:
I just added it as I was asked to do so the last time. But from my side we can also put a [German] in front.

Reply:
Please put it to the end. I think that makes reading easier.

Reply:
Created a poll to discuss: https://nuudel.digitalcourage.de/rCEGG1Jc3CIkDq1W

Reply:
I decided to go for the language indicator after a link. If this does not apply to be useful, we can still change. Now let's see how it goes for a while!